docs/tutorials/onnx/inference_on_onnx_model.md

```diff
@@ -75,7 +75,7 @@ Create the model folder and download the zipped model
 
 
 ```python
-os.makedirs(model_folder, exist_ok=True)
```

Comment:
can you add:
`if os.path.isdir(model_folder):`
otherwise you get:
```
OSErrorTraceback (most recent call last)
<ipython-input-17-5b2c412acdc0> in <module>()
----> 1 os.makedirs(model_folder)
      2 if not os.path.isfile(archive_file):
      3     wget.download(url, model_folder)

/home/ubuntu/anaconda3/envs/mxnet_p27/lib/python2.7/os.pyc in makedirs(name, mode)
    155         if tail == curdir:           # xxx/newdir/. exists if xxx/newdir exists
    156             return
--> 157     mkdir(name, mode)
    158 
    159 def removedirs(name):

OSError: [Errno 17] File exists: 'model'
```

Reply:
you mean  :  if  not os.path.isdir(model_folder):